pages/preprints.py

```diff
@@ -113,13 +117,12 @@ def select_from_dropdown_listbox(self, selection):
     )
 
     def select_top_level_subject(self, selection):
+        subject_selector = 'div[data-analytics-scope="Browse"] > ul > li'
+        wait = WebDriverWait(self.driver, 20)
+        wait.until(text_to_be_present_in_elements((By.CSS_SELECTOR, subject_selector), selection))
```

Comment:
This might be fine, but isn't there a selector to check text contents? I'm pretty sure there's one for xpath, not sure about css.